vsketch/vsketch.py

```diff
@@ -881,6 +881,10 @@ def polygon(
 
                 >>> vsk = Vsketch()
                 >>> vsk.polygon([(0, 0), (2, 3), (3, 2)])
+            
+            A single complex iterable of size 1 can also be used::
+
+                >>> vsk.polygon(np.array([3 + 3j, 2 + 5j, 4 + 7j]))
```

Comment:
A nouveau, pour l'example de base, vire le np.array. De plus, je dirais que `size 1` porte à confusion. Je dirais `A 1-dimension utterable fo complex can also be used`).

Reply:
C'est complété ! :)